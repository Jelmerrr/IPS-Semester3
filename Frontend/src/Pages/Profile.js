
import React, { useState, useEffect } from "react";

const ProfileName = "Hoi";

function Profile() {
    return (
        <div>
            <h1>Profile page</h1>
            <h1>{ProfileName}</h1>
        </div>
    );
}
export default Profile;
