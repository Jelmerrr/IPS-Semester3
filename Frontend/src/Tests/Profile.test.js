import renderer from 'react-test-renderer';
import Profile from '../Pages/Profile';
import React from "react";

it('renders correctly', () => {
    const tree = renderer
        .create(<Profile page="http://www.test.com">Test</Profile>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});