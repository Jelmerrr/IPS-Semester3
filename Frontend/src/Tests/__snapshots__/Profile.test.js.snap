// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Profile page
  </h1>
  <h1>
    Hoi
  </h1>
</div>
`;
